docs/getting_started.rst

```diff
@@ -77,6 +77,47 @@ on if a server goes down.
     client.set('some_key', 'some value')
     result = client.get('some_key')
 
+Key distribution is handled by the ``hasher`` argument in the constructor. The
+default is the built-in :class:`pymemcache.client.rendezvous.RendezvousHash`
+hasher. It uses the built-in :class:`pymemcache.client.murmur3.murmur3_32`
+implementation to distribute keys on servers. Overriding these two parts can be
+used to change how keys are distributed. Changing the hashing algorithm can be
+done by setting the ``hash_function`` argument in the ``RendezvousHash``
+constructor.
+
+Rebalancing in the :class:`pymemcache.client.hash.HashClient` functions as'
```

Comment:
Stray apostrophe?

```suggestion
Rebalancing in the :class:`pymemcache.client.hash.HashClient` functions as
```

Reply:
Oh yeah, my bad. Thought I proofread it. Good catch. :+1:

Reply:
Hi @martinnj ,
The hashclient tries to ``get("key2")`` but detects the node as down. This causes it to mark the node as down. Removing it from the hasher. The hasclient can attempt to retry the operation based on the ``retry_attempts`` and ``retry_timeout`` arguments.
In the above point will it  retry the operation or client?

Reply:
@glory-123 I don't think this PR is the place for that question, but the docstring of the class should hgopefully clarify the behavior in question:
https://github.com/pinterest/pymemcache/blob/master/pymemcache/client/hash.py#L66-L71
It will attempt the operation `retry_attempts` times with `retry_timeout` time in between before marking the node as dead for `dead_timeout` seconds.